lib/rails_data_migrations/migrator.rb

```diff
@@ -38,12 +38,18 @@ def migrations_path
         'db/data_migrations'
       end
 
+      def rails_6_0?
+        Rails::VERSION::MAJOR > 6 || (Rails::VERSION::MAJOR == 6 && Rails::VERSION::MINOR >= 0)
```

Comment:
I assume `Rails::VERSION::MAJOR > 6` could be omitted from conditional?

Reply:
Good question! I modeled it after the existing `rails_5_2?` method. I suppose the `Rails::VERSION::MAJOR > 6` is to future proof it for upcoming versions of Rails, otherwise Rails 7+ would not work out-of-the-box. 

I'm happy to remove it if you think that's best.

Reply:
Existing conditional for rails 5 filters out rails 5.0 and 5.1

For the rails 6 it could be simpler. Given your reasonable notice on future versions I would simplify it to just `>=6`. What do you think?

Reply:
Ahh gotcha. That makes perfect sense!

Yes `>=6` looks like the way to go. I can make that change and update the PR tomorrow if you’d like.